dns-policy-operator/app/policy/management/commands/create_reviewer.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Create reviewer command."""
+
+import getpass
+
+from django.contrib.auth.models import Group, User
+from django.contrib.auth.password_validation import validate_password
+from django.core import exceptions
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Create reviewer command."""
+
+    help = 'Create a new reviewer'
+
+    def add_arguments(self, parser):
+        """Define CLI arguments."""
+        parser.add_argument('username', type=str, help='Username of the user')
+        parser.add_argument('email', type=str, help='Email of the user')
+
+    def handle(self, *args, **options):
+        """Handle CLI invocation."""
+        username = options['username']
+        email = options['email']
+
+        # Took example on Django here
+        # ref: https://github.com/django/django/blob/main/django/contrib/auth/management/commands/createsuperuser.py#L172
+        password = getpass.getpass()
```

Comment:
Shouldn't we provide a way to pass the password non interactively ? For example through envvar or through stdin ?
If getpass supports stdin, shouldn't we have some logic to avoid repeating the password in such cases ?

Reply:
I added a way to pass the password using STDIN